packages/graphql/tests/integration/directives/populatedBy.int.test.ts

```diff
@@ -1465,6 +1465,972 @@ describe("@populatedBy directive", () => {
+                                callback,
+                            },
+                        },
+                    },
+                });
+
+                const movieId = generate({
+                    charset: "alphabetic",
+                });
+
+                const mutation = `
+                    mutation {
+                        ${testMovie.operations.update}(where: { id: "${movieId}" }, update: { id: "${movieId}" }) {
+                            ${testMovie.plural} {
+                                id
+                                callback
+                            }
+                        }
+                    }
+                `;
+
+                await testHelper.executeCypher(`
+                        CREATE (:${testMovie.name} { id: "${movieId}" })
+                    `);
+
+                const result = await testHelper.executeGraphQL(mutation);
+
+                expect(result.errors).toBeUndefined();
+                expect(result.data as any).toMatchObject({
+                    [testMovie.operations.update]: {
+                        [testMovie.plural]: [
+                            {
+                                id: movieId,
+                                callback: date.toISOString(),
+                            },
+                        ],
+                    },
+                });
+            });
+
+            test("should throw an error if string is not a DateTime", async () => {
+                const testMovie = testHelper.createUniqueType("Movie");
+
+                const callback = () => Promise.resolve("banana");
+
+                const typeDefs = gql`
+                    type ${testMovie.name} {
+                        id: ID
+                        callback: DateTime! @populatedBy(operations: [CREATE], callback: "callback")
+                    }
+                `;
+
+                await testHelper.initNeo4jGraphQL({
+                    typeDefs,
+                    features: {
+                        populatedBy: {
+                            callbacks: {
+                                callback,
+                            },
+                        },
+                    },
+                });
+
+                const movieId = generate({
+                    charset: "alphabetic",
+                });
+
+                const mutation = `
+                    mutation {
+                        ${testMovie.operations.create}(input: [{ id: "${movieId}" }]) {
+                            ${testMovie.plural} {
+                                id
+                                callback
+                            }
+                        }
+                    }
+                `;
+
+                const result = await testHelper.executeGraphQL(mutation);
+
+                expect(result.errors).toHaveLength(1);
+                expect(result.errors?.[0]?.message).toBe("DateTime cannot represent non temporal value: banana");
```

Comment:
I would consider doing a single expect check on errors

```suggestion
                expect(result.errors).toEqual(["DateTime cannot represent non temporal value: banana"]);
```

Reply:
We can't just use `toEqual` here because the response is an object containing a `message` property. We could this instead, though it is probably even more verbose than the original:

```ts
expect(result.errors).toEqual(
    expect.arrayContaining([
        expect.objectContaining({
            message: "DateTime cannot represent non temporal value: banana",
        }),
    ])
);
```